# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xorg/linux-joystick_h.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -dur xc/programs/Xserver/hw/xfree86/input/ur98/xf86Ur-98.c src.xorg.1129808283.2035.3041395841/xc/programs/Xserver/hw/xfree86/input/ur98/xf86Ur-98.c
--- xc/programs/Xserver/hw/xfree86/input/ur98/xf86Ur-98.c	2004-04-23 21:54:05.000000000 +0200
+++ src.xorg.1129808283.2035.3041395841/xc/programs/Xserver/hw/xfree86/input/ur98/xf86Ur-98.c	2005-10-20 21:27:44.000000000 +0200
@@ -5,8 +5,18 @@
 #include <string.h>
 #include <errno.h>
 #define inline __inline__
+
+#ifdef __STRICT_ANSI__
+#define REDEFINE__STRICT_ANSI__
+#undef __STRICT_ANSI__
+#endif
+
 #include <linux/joystick.h>
 
+#ifdef REDEFINE__STRICT_ANSI__
+#define __STRICT_ANSI__
+#endif
+
 #undef BUS_ISA
 #undef BUS_PCI
 
